checker/jdk/index/src/java/lang/String.java

```diff
@@ -163,7 +163,7 @@ public String() {
      * @param  original
      *         A {@code String}
      */
-    public /*!PolyMinLen*/ @PolySameLen String(/*!PolyMinLen*/ @PolySameLen String original) {
+    public @PolyValue @PolySameLen String(@PolyValue @PolySameLen String original) {
```

Comment:
Technically these are Value Checker annotations, so they don't belong in the index annotated jdk. They should go in org/checkerframework/common/value/statically-executable.astub.  (Could you also move the MinLen annotations there, too?)